File: daily_challenges/3228.py
class Solution:
    def maxOperations(self, s: str) -> int:
        n = len(s)
        result = 0
        i = 0
        count1 = 0
        while i < n:
            if s[i] == "0":
                result += count1
                while i < n and s[i] == "0":
                    i += 1
            else:
                count1 += 1
                i += 1
        return result

File: daily_challenges/3228.js
/**
 * @param {string} s
 * @return {number}
 */
var maxOperations = function (s) {
    const n = s.length
    let result = 0
    let i = 0
    let count1 = 0
    while (i < n) {
        if (s[i] === '0') {
            result += count1
            while (i < n && s[i] === '0') {
                i++
            }
        } else {
            count1++
            i++
        }
    }
    return result
};